data/map planets.txt

```diff
@@ -3735,6 +3807,14 @@ planet "Station Cian"
 	"required reputation" 15
 	security 0.5
 
+planet Steamwater
+	attributes "gas giant" "requires: gaslining" uninhabited
+	landscape land/clouds_05
+	description `Steamwater is a hot Neptune, a misleadingly named "ice giant" close to its parent star. Steamwater is distinctive in that, by mass, it is almost entirely water. Beyond this, the range of temperatures and pressures found in its atmosphere is such that its thickest layer is composed of supercritical water, a phase state blending together the states of vapor and liquid, making flight inside it rather disorienting.`
+	description `	With no true interface between gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your altimeter. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`
```

Comment:
It couldn't actually be an altimeter, no existing form of altimeter would be able to handle buoyancy effects or an inconclusive phase state boundary.
Inertial navigation would use gyroscopes and accelerometers to track where you're going relative to where you were.
```suggestion
	description `	With no true interface between gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your inertial navigation aid. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`
```

Reply:
Yeah, but this isn't some fancy future tech. Inertial navigation is technology we've been using for many years already in the real world.

Reply:
As an average person, I would understand what an "altimeter" is referring to and that sentence makes sense to me. 
"inertial navigation aid" sounds like nonsense which my eyes just kinda glaze over and the rest of the sentence doesn't make sense as a result.

Reply:
I suppose in a theoretical future with space travel, an "altimeter" could have become a device that incorporates such things.
Given the type of player who's actually likely to fly around in a gaslined ship and find this though, I'd suggest still at least calling it an "inertial altimeter".
```suggestion
	description `	With no true interface between gaseous water vapor and the compressed liquid found below, the buoyant force affecting your ship is unpredictable without keeping a careful eye on your inertial altimeter. While this is true of gas giants in general, the much higher density of water as compared to common hydrogen makes this effect especially noticeable.`
```